### PR TITLE
bulk account loader

### PR DESCRIPTION
**Pull request by crispheaney**

### CODE REVIEW

File: src/driftpy/accounts/bulk_account_loader.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            rpc_request = jsonrpcclient.request(
+                "getMultipleAccounts",
+                params=[
+                    pubkeys_to_send,
+                    {"encoding": "base64", "commitment": self.commitment},
+                ],
+            )
+            rpc_requests.append(rpc_request)
+
+        resp = await self.connection._provider.session.post(
```
Comment (crispheaney) [inline src/driftpy/accounts/bulk_account_loader.py:126]: should add time out

File: src/driftpy/accounts/types.py [hunk 2]
```diff
@@ -40,13 +48,21 @@ async def get_spot_market_and_slot(
...
+    @abstractmethod
+    async def subscribe(self):
+        pass
+
+    @abstractmethod
+    def unsubscribe(self):
+        pass
+
     @abstractmethod
     async def get_user_account_and_slot(self) -> Optional[DataAndSlot[User]]:
```
Comment (crispheaney) [inline src/driftpy/accounts/types.py:67]: should i make these not async?

File: src/driftpy/drift_client.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def __init__(
         self.subaccounts = [0]
 
         if account_subscriber is None:
-            account_subscriber = CachedDriftClientAccountSubscriber(self.program)
+            account_subscriber = WebsocketDriftClientAccountSubscriber(self.program)
```
Comment (0xbigz) [inline src/driftpy/drift_client.py:75]: this is default to websocket? is that same as typescript

File: src/driftpy/accounts/oracle.py [hunk 4]
```diff
@@ -73,9 +57,26 @@ async def _parse_pyth_price_info(
...
-        last_slot,
-        twac,
-        twap,
+    pyth_price_info = PythPriceInfo.deserialise(buffer, offset, exponent=exponent)
+
+    scale = 1
+    if "1K" in str(oracle_source):
+        scale = 1e3
+    elif "1M" in str(oracle_source):
+        scale = 1e6
```
Comment (0xbigz) [inline src/driftpy/accounts/oracle.py:66]: todo: add support for the stable type

Comment (crispheaney) [reply src/driftpy/accounts/oracle.py:66]: yeah and there is an issue with Pyth versus PYTH

File: src/driftpy/drift_client.py (hunk 3 above)
Comment (crispheaney) [reply src/driftpy/drift_client.py:75]: yeah

**Pull request merged**